addons/mail/controllers/attachment.py

```diff
@@ -141,7 +144,7 @@ def mail_attachement_update_thumbnail(self, attachment_id, thumbnail, access_tok
             raise request.not_found()
         # sudo: ir.attachment: access check is done above, sudo necessary for guests
         attachment_sudo = attachment.sudo()
```

Comment:
Note: while I'm seeing it, we might want to add an extra check on the route, to ensure the target attachment is indeed a pdf (check mimetype) to prevent users from being able to upload thumbnail for other kind of files.

Reply:
Done!